src/app/static/src/app/store/dataExploration/getters.ts

```diff
@@ -0,0 +1,34 @@
+import {Getter, GetterTree} from "vuex";
+import {Error} from "../../generated"
+import {extractErrors} from "../../utils";
+import {DataExplorationState} from "./dataExploration";
+
+interface DataExplorationGetters {
+    isGuest: Getter<DataExplorationState, DataExplorationState>
+    errors: Getter<DataExplorationState, DataExplorationState>
+}
+
+export const getters: DataExplorationGetters & GetterTree<DataExplorationState, DataExplorationState> = {
+    isGuest: (state: DataExplorationState) => {
```

Comment:
this is duplicated from root getters, but is so small as to probably not be worth pulling into it's own shared module